backend/hitas/views/apartment_max_price.py

```diff
@@ -123,6 +128,28 @@ def update(self, request, *args, **kwargs):
             ampc.delete()
             return Response(status=HTTPStatus.NO_CONTENT)
 
+    @action(detail=True, methods=["GET"])
+    def download(self, request, **kwargs) -> HttpResponse:
+        with model_or_404(ApartmentMaximumPriceCalculation):
+            mpc = ApartmentMaximumPriceCalculation.objects.get(uuid=kwargs["pk"])
+
+            # Make sure the calculation is confirmed
+            if mpc.confirmed_at is None:
```

Comment:
how is unconfirmed max price downloaded?

Reply:
It's not yet, that will be done as a separate task from this

Reply:
that's fine. just thinking we don't need to define a new error response here. This could just always assume the calculation is confirmed?

Reply:
As the PDF is going to end up the wording "confirmed" ("vahvistettu"), i'd rather just block generating PDFs from unconfirmed "more in depth" calculations. IIRC there is never going to be a case when a PDF is needed for this "more in depth" calculation that isn't confirmed. (Unconfirmed max price PDF values are generated similarly to how the unconfirmed prices are on Apartment details page)